code/datums/abilities/flock/flockmind.dm

```diff
@@ -167,12 +167,42 @@
 
 	logTheThing(LOG_COMBAT, holder.get_controlling_mob(), "designates [constructTarget(M)] as [flock.isEnemy(M) ? "" : "not "]an enemy at [log_loc(src.holder.owner)].")
 
+	flock.removeAlly(target)
+
 	if (flock.isEnemy(M))
 		flock.removeEnemy(M)
 		return
 
 	flock.updateEnemy(M)
 
+
+/datum/targetable/flockmindAbility/designateAlly
```

Comment:
Adds another ability to the big list of abilities. What if the Designate Enemy ability was more of a Designate Enemy Status ability, where clicking on someone with it will cycle through Enemy/Ally/None, could also have it be sticky

Reply:
it is yet another ability, but I don't see a better way, cycling through them sounds like a good way to accidentally have your turrets/drones instantly murder someone you were trying to set to ally